src/main/kotlin/com/vaadin/plugin/copilot/service/CopilotServerServiceImpl.kt

```diff
@@ -13,6 +16,8 @@ import java.util.*
 // Server implementation based on https://github.com/teocci/NioSocketCodeSample/tree/master
 class CopilotServerServiceImpl(private val project: Project): CopilotServerService {
 
+    private val LOG: Logger = Logger.getInstance(CopilotPluginUtil::class.java)
```

Comment:
Should not this be?:
```suggestion
    private val LOG: Logger = Logger.getInstance(CopilotServerServiceImpl::class.java)
```

Seems like pasted and the class name was not rewritten, not a big thingie, log entry will contain the other class name not this one.

Reply:
No, by purpose I am using same logger to easily allow disabling it by single rule

Reply:
Maybe it should more generic name, like just "Copilot"

Reply:
Ah, okay, Copilot or Copilot-Plugin, okay, I see! Then we can leave it as it is.